docs/integrations/sources/stripe-migrations.md

```diff
@@ -1,5 +1,16 @@
 # Stripe Migration Guide
 
+## Upgrading to 5.0.0
+
+This change fixes multiple issues for the `Refunds`, `Checkout Sessions` and `Checkout Sessions Line Items` streams:
+
+ - `CheckoutSessions` stream missing data for one day when using the incremental sync mode after a reset
+ - `Refunds` stream not syncing data in the incremental sync mode
```

Comment:
You also mentioned in [your comment](https://github.com/airbytehq/airbyte/pull/32286#issuecomment-1808388516) that `To always have the up-to-date data, customers can make use of the lookback window`. Should explicit the cases where it might be useful for a user to configure the lookback window?

Reply:
Done